charts/saml-proxy/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: saml-proxy
 description: A Helm chart for Kubernetes
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
The `pipeline.js` will update this version automatically, the version number is just a placeholder as the version is determined by semantic release package, could you please revert it back to `0.1.0`? Thanks

Reply:
Thanks. I've reverted the version number.

Reply:
Great, thanks!